lang/po/Firmware_fr.po

```diff
@@ -163,13 +163,13 @@ msgid ""
 "Bed leveling failed. Sensor didn't trigger. Debris on nozzle? Waiting for "
 "reset."
 msgstr ""
-"Capt. non declenche pendant que je nivele le plateau. Debris sur buse? En "
+"Capt. non déclenche pendant que je nivelé le lit. Debris sur buse? En "
```

Comment:
A series of non correct accents here, please find a proposal here : 

Capt. non déclenché pendant que je nivèle le plateau. Débris sur buse?

Reply:
It is already there https://github.com/3d-gussner/Prusa-Firmware/blob/MK3_French/lang/po/Firmware_fr.po#L166